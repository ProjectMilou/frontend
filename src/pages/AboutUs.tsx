import React from 'react';
import {
  Box,
  Typography,
  Grid,
  makeStyles,
  createStyles,
  Theme,
  useTheme,
} from '@material-ui/core';
import { RouteComponentProps } from '@reach/router';
import ProfileCard from '../components/shell/landingPage/ProfileCard';
import LogoCard from '../components/shell/landingPage/LogoCard';

import linda from '../assets/images/linda.png';
import chris from '../assets/images/chris.png';
import julien from '../assets/images/julien.png';

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    boxClass: {
      paddingInline: theme.spacing(13),
      maxWidth: theme.spacing(175),
      margin: 'auto',
    },
  })
);

const AboutUs: React.FC<RouteComponentProps> = () => {
  const theme = useTheme();
  const { boxClass } = useStyles(theme);

  return (
    <Box>
      <Box pt={4} pb={6}>
        <Grid container direction="column" alignItems="center">
          <Grid item>
            <Typography
              variant="h3"
              align="center"
              style={{ marginTop: '16px', marginBottom: '16px' }}
            >
              Our Team
            </Typography>
          </Grid>
          <Grid item>
            <Typography variant="body1" align="center" className={boxClass}>
              Apps like Trade Republic and other direct brokers make investing
              easier and easier. The stock market is now accessible to everyone.
              Even so, many individuals do not have access to tools to make
              informed and informed investment decisions. Current solutions are
              often confusing and tailored to professional investors. This is
              exactly where we attack.
            </Typography>
          </Grid>
        </Grid>
      </Box>
      <Box
        bgcolor="#0c1a3a"
        color="white"
        py={12}
        m="auto"
        style={{
          display: 'flex',
          justifyContent: 'center',
        }}
      >
        <Typography variant="body1" align="center" className={boxClass}>
          Apps like Trade Republic and other direct brokers make investing
          easier and easier. The stock market is now accessible to everyone.
          Even so, many individuals do not have access to tools to make informed
          and informed investment decisions. Current solutions are often
          confusing and tailored to professional investors. This is exactly
          where we attack.
        </Typography>
      </Box>
      <Box className={boxClass}>
        <Grid
          container
          direction="row"
          justify="space-evenly"
          alignItems="center"
        >
          <Grid item>
            <ProfileCard
              name="Linda"
              title="CO-FOUNDER"
              content="Ich bin Social Entrepreneur. "
              img={linda}
            />
          </Grid>
          <Grid item>
            <ProfileCard
              name="Chris"
              title="CO-FOUNDER"
              content="Ich bin Networker."
              img={chris}
            />
          </Grid>
          <Grid item>
            <ProfileCard
              name="Julien"
              title="CO-FOUNDER"
              content="Ich bin Stratege."
              img={julien}
            />
          </Grid>
        </Grid>
      </Box>
      <LogoCard />
    </Box>
  );
};

export default AboutUs;
