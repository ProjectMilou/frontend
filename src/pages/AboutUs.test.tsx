import { render } from '@testing-library/react';
import * as React from 'react';
import AboutUs from './AboutUs';

describe('AboutUs', () => {
  test('should render aboutus-page', async () => {
    const { container } = render(<AboutUs />);
    expect(container).toMatchSnapshot();
  });
});
