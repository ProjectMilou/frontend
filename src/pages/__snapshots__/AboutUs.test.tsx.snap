// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs should render aboutus-page 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-2"
  >
    <div
      class="MuiBox-root MuiBox-root-3"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
      >
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter"
            style="margin-top: 16px; margin-bottom: 16px;"
          >
            Our Team
          </h3>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <p
            class="MuiTypography-root makeStyles-boxClass-1 MuiTypography-body1 MuiTypography-alignCenter"
          >
            Apps like Trade Republic and other direct brokers make investing easier and easier. The stock market is now accessible to everyone. Even so, many individuals do not have access to tools to make informed and informed investment decisions. Current solutions are often confusing and tailored to professional investors. This is exactly where we attack.
          </p>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-4"
      style="display: flex; justify-content: center;"
    >
      <p
        class="MuiTypography-root makeStyles-boxClass-1 MuiTypography-body1 MuiTypography-alignCenter"
      >
        Apps like Trade Republic and other direct brokers make investing easier and easier. The stock market is now accessible to everyone. Even so, many individuals do not have access to tools to make informed and informed investment decisions. Current solutions are often confusing and tailored to professional investors. This is exactly where we attack.
      </p>
    </div>
    <div
      class="MuiBox-root MuiBox-root-5 makeStyles-boxClass-1"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-evenly"
      >
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <div
            class="MuiBox-root MuiBox-root-6"
            justify-content="center"
          >
            <div
              class="MuiAvatar-root MuiAvatar-circle"
              style="width: 230px; height: 230px;"
            >
              <img
                class="MuiAvatar-img"
                src="linda.png"
              />
            </div>
            <h3
              class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter"
            >
              Linda
            </h3>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-alignCenter"
            >
              CO-FOUNDER
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
            >
              Ich bin Social Entrepreneur. 
            </p>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
            >
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  href="https://www.linkedin.com/linda-karger"
                  tabindex="0"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                        />
                      </svg>
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
              <div
                class="MuiGrid-root"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  href="mailto:Linda.karger@manageandmore.de"
                  tabindex="0"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                        />
                      </svg>
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <div
            class="MuiBox-root MuiBox-root-7"
            justify-content="center"
          >
            <div
              class="MuiAvatar-root MuiAvatar-circle"
              style="width: 230px; height: 230px;"
            >
              <img
                class="MuiAvatar-img"
                src="chris.png"
              />
            </div>
            <h3
              class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter"
            >
              Chris
            </h3>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-alignCenter"
            >
              CO-FOUNDER
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
            >
              Ich bin Networker.
            </p>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
            >
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  href="https://www.linkedin.com/christoph-hanus"
                  tabindex="0"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                        />
                      </svg>
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
              <div
                class="MuiGrid-root"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  href="mailto:Christoph.hanus@manageandmore.de"
                  tabindex="0"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                        />
                      </svg>
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <div
            class="MuiBox-root MuiBox-root-8"
            justify-content="center"
          >
            <div
              class="MuiAvatar-root MuiAvatar-circle"
              style="width: 230px; height: 230px;"
            >
              <img
                class="MuiAvatar-img"
                src="julien.png"
              />
            </div>
            <h3
              class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter"
            >
              Julien
            </h3>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-alignCenter"
            >
              CO-FOUNDER
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
            >
              Ich bin Stratege.
            </p>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
            >
              <div
                class="MuiGrid-root MuiGrid-item"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  href="https://www.linkedin.com/schmidt-julien"
                  tabindex="0"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                        />
                      </svg>
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
              <div
                class="MuiGrid-root"
              >
                <a
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiButton-root MuiButton-text"
                  href="mailto:Julien.Schmidt@manageandmore.de"
                  tabindex="0"
                >
                  <span
                    class="MuiButton-label"
                  >
                    <span
                      class="MuiButton-startIcon MuiButton-iconSizeMedium"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"
                        />
                      </svg>
                    </span>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-10"
    >
      <div
        class="MuiGrid-root MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-grid-xs-true"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3"
          >
            Our supporters
          </h3>
          <p
            class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
          >
            Thank you for your help!
          </p>
        </div>
      </div>
      <div
        class="MuiBox-root MuiBox-root-11"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
        >
          <ul
            class="MuiGridList-root"
            style="margin: -9px;"
          >
            <li
              class="MuiGridListTile-root"
              style="width: 16.666666666666668%; height: 138px; padding: 9px;"
            >
              <div
                class="MuiGridListTile-tile"
              >
                <img
                  class="MuiCardMedia-root makeStyles-logo-9 MuiCardMedia-media MuiCardMedia-img"
                  src="180.png"
                />
              </div>
            </li>
            <li
              class="MuiGridListTile-root"
              style="width: 16.666666666666668%; height: 138px; padding: 9px;"
            >
              <div
                class="MuiGridListTile-tile"
              >
                <img
                  class="MuiCardMedia-root makeStyles-logo-9 MuiCardMedia-media MuiCardMedia-img"
                  src="getThumb.gif"
                />
              </div>
            </li>
            <li
              class="MuiGridListTile-root"
              style="width: 16.666666666666668%; height: 138px; padding: 9px;"
            >
              <div
                class="MuiGridListTile-tile"
              >
                <img
                  class="MuiCardMedia-root makeStyles-logo-9 MuiCardMedia-media MuiCardMedia-img"
                  src="tumlogo.png"
                />
              </div>
            </li>
            <li
              class="MuiGridListTile-root"
              style="width: 16.666666666666668%; height: 138px; padding: 9px;"
            >
              <div
                class="MuiGridListTile-tile"
              >
                <img
                  class="MuiCardMedia-root makeStyles-logo-9 MuiCardMedia-media MuiCardMedia-img"
                  src="manage.png"
                />
              </div>
            </li>
            <li
              class="MuiGridListTile-root"
              style="width: 16.666666666666668%; height: 138px; padding: 9px;"
            >
              <div
                class="MuiGridListTile-tile"
              >
                <img
                  class="MuiCardMedia-root makeStyles-logo-9 MuiCardMedia-media MuiCardMedia-img"
                  src="tech-talents.png"
                />
              </div>
            </li>
            <li
              class="MuiGridListTile-root"
              style="width: 16.666666666666668%; height: 138px; padding: 9px;"
            >
              <div
                class="MuiGridListTile-tile"
              >
                <img
                  class="MuiCardMedia-root makeStyles-logo-9 MuiCardMedia-media MuiCardMedia-img"
                  src="start.png"
                />
              </div>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
